.github/workflows/run-circle-mlir-build.yml

```
@@ -0,0 +1,66 @@
+name: Run Circle-MLIR Ubuntu Build
+
+on:
+  push:
+    branches:
+      - master
+      - release/*
+    paths:
+      - '.github/workflows/run-circle-mlir-build.yml'
+      - 'circle-mlir/**'
+  pull_request:
+    branches:
+      - master
+      - release/*
+    paths:
+      - '.github/workflows/run-circle-mlir-build.yml'
+      - 'circle-mlir/**'
+
+defaults:
+  run:
+    shell: bash
+
+# Cancel previous running jobs when pull request is updated
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  circle-mlir-test:
+    if: github.repository_owner == 'Samsung'
+    strategy:
+      matrix:
+        type: [ Debug, Release ]
+        ubuntu_code: [ jammy ]
+        include:
+          - ubuntu_code: jammy
+            ubuntu_vstr: u2204
+
+    runs-on: ubuntu-24.04
```

```suggestion
    runs-on: ubuntu-latest
```

Looking at the other workflows, `ubuntu-24.04`, `ubuntu-22.04`, `ubuntu-20.04`, and `ubuntu-latest` versions are being used interchangeably.
How about standardizing to `ubuntu-latest` in this workflow?

---

Never tried this, so we can try now :)